src/NativeModules/UIManager.js

```diff
@@ -61,6 +61,11 @@ const UIManager = {
   configureNextLayoutAnimation(callback, errorCallback) {
 
   },
+  AndroidDrawerLayout: {
+    Constants: {
+      DrawerPosition: { Left: 8388611, Right: 8388613 },
```

Comment:
Where are you getting these numbers from. Not sure I like the use of _magic numbers_.